src/main/java/de/tum/in/www1/artemis/repository/StatisticsRepository.java

```diff
@@ -114,4 +205,343 @@
             order by r.completionDate
             """)
     List<Map<String, Object>> getResultFeedbacks(@Param("startDate") ZonedDateTime startDate, @Param("endDate") ZonedDateTime endDate);
+
+    @Query("""
+            select r.completionDate as day, sum(size(r.feedbacks)) as amount
+            from Result r
+            where r.completionDate >= :#{#startDate} and r.completionDate <= :#{#endDate} and r.participation.exercise.id in :exerciseIds
+            group by r.completionDate
+            order by r.completionDate
+            """)
+    List<Map<String, Object>> getResultFeedbacksForCourse(@Param("startDate") ZonedDateTime startDate, @Param("endDate") ZonedDateTime endDate,
```

Comment:
please use a custom object here and not a map

Reply:
Done